modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/db/Statement.scala

```diff
@@ -111,8 +111,11 @@ object Statement {
       val frRegion = Fragment.const0(region)
       val frMaxError = Fragment.const0(maxError.toString)
       val frCompression = getCompressionFormat(compression)
+      val frColumns = Fragment.const0(TableDefinitions.atomicColumns.map(_.columnName).mkString(","))
```

Comment:
I am not sure about this usage because even though I tried to copy the exact table definition, this still can be different from the actual event table definition. I think better way is getting the column names in the beginning and passing it to here.

Reply:
I think the chance somebody is missing columns is abysmall. We have this table structure for ages and if somebody is not up-to-date yet - they likely cannot use the rest of our apps. The last change we've made was in 2015: https://github.com/snowplow/snowplow/commits/master/4-storage/redshift-storage/sql/atomic-def.sql

However, you also can add a test like this one in Snowflake Loader: https://github.com/snowplow-incubator/snowplow-snowflake-loader/blob/0037c34b5f8c393ebae27216e770cbd27bf53e53/loader/src/test/scala/com/snowplowanalytics/snowflake/loader/ast/AtomicDefSpec.scala#L23-L32. To make sure `TableDefinitions.atomicColumns` always represent sensible structure.